test/e2e/network/netpol/network_policy_api.go

```diff
@@ -209,4 +198,62 @@ var _ = common.SIGDescribe("Netpol API", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(len(nps.Items), 0, "filtered list should have 0 items")
 	})
+
+	/*
+		Release: v1.21
+		Testname: NetworkPolicy support EndPort Field
+		Description:
+		- EndPort field cannot be defined if the Port field is not defined
+		- EndPort field cannot be defined if the Port field is defined as a named (string) port.
+		- EndPort field must be equal or greater than port.
+	*/
+	ginkgo.It("should support creating NetworkPolicy API with endport field [Feature:NetworkPolicyEndPort]", func() {
```

Comment:
I think the standard here is to put the Feature on the beginning of the line, like:
```suggestion
	ginkgo.It("[Feature:NetworkPolicyEndPort] should support creating NetworkPolicy API with endport field", func() {
```

Reply:
I see other case in netpol folder, they all put the feature label on the end of the line.
like:
https://github.com/kubernetes/kubernetes/blob/b4ea4322a660717084dcd09b26fe27e55ebb39f0/test/e2e/network/netpol/network_legacy.go#L161

https://github.com/kubernetes/kubernetes/blob/b4ea4322a660717084dcd09b26fe27e55ebb39f0/test/e2e/network/netpol/network_policy.go#L181